buffer/src/main/java/io/netty/buffer/AbstractAllocatorEvent.java

```diff
@@ -27,6 +28,7 @@
 @Category("Netty")
 @SuppressWarnings("Since15")
 abstract class AbstractAllocatorEvent extends Event {
+    @Label("Allocator type")
     @Description("The type of allocator this event is for")
-    public AllocatorType allocatorType;
+    public String allocatorType;
```

Comment:
i'm not sure but this might be substantially less storage efficient.

Reply:
We could put a class reference instead, to the allocator implementation? Fact is that enums don't work, because JFR ignores them, turns out.

Reply:
a `byte` would probably be most efficient. but a Class might make more sense semantically.